src/clients/RTorrent.ts

```diff
@@ -29,14 +29,19 @@ async function createLibTorrentResumeTree(
 	async function getFileResumeData(
 		file: File
 	): Promise<LibTorrentResumeFileEntry> {
-		const filePath = resolve(dataDir, file.path);
+		let filePath = file.path;
+		if (filePath.indexOf(sep) != -1) {
+			filePath = filePath.split(sep).slice(1).join(sep);
+		}
+
+		const normalizedFilePath = resolve(dataDir, filePath);
```

Comment:
Forgive my ignorance, I don't have this setup to test this myself, but does this not make an assumption that the path is at a depth that can't be guaranteed (1)

Does this function as intended if I used (for example) "/media/downloads/torrents" as my default directory for downloads?

Reply:
This function tries to split a filename inside a torrent (for torrents with folders). For example, if you have `BLURAY_DISC1/BDMV/index.json` file inside the torrent, it would slice the top folder and filePath would be `BDMV/index.json`.

It then adds the dataDir, in your example `/media/download/torrents/SOME_OTHER_PATH_WE_CHOSE_TO_MOVE_IT`.
The final checked path would be: `/media/download/torrents/SOME_OTHER_PATH_WE_CHOSE_TO_MOVE_IT/BDMV/index.json`

My example is for a renamed torrent, but in the case the torrent has its normal name and was not renamed, it would check: `/media/download/torrents/BLURAY_DISC1/BDMV/index.json` path.